testing/kuttl/e2e/password-change/03-psql-connect-uri.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: psql-connect-uri2
+spec:
+  backoffLimit: 3
+  template:
+    spec:
+      restartPolicy: "OnFailure"
+      containers:
+        - name: psql
+          image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.1-0
+          command:
+            - psql
+            - "$(PGURI)"
+            - -c
+            - "select version();"
+          env:
+          - name: PGURI
+            valueFrom: { secretKeyRef: { name: password-change-pguser-password-change, key: uri } }
```

Comment:
🤔 It might be possible to use hardcoded `PGPASSWORD` in these URI files, too.

Reply:
Doesn't the URI test implicitly check that the password has been updated and that the reconcile has updated the URI with the new password?

Reply:
I see, yes. Perhaps a comment here that the prior step waited for PGO to update the URI?

After the secret is updated and before PGO acts, the secret contains a `password` and `uri` that disagree. This step only looks at `uri` and would pass during that time. After PGO acts, this step checks PGO's work in Postgres and on the `uri` field.